Assets/Scripts/PlayerAttack.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerAttack : MonoBehaviour
+{
+    private float timeAttackCD;
+    public float startAttacktime;
```

Comment:
make some of these private if you aren't intending to change them a lot while testing.

Reply:
Sounds good